types/jscodeshift/src/Runner.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Options } from './core';
+
+export function run(
+    transformFile: string,
+    paths: string[],
+    options?: Options,
+): Promise<{
+    stats: string;
+    timeElapsed: string;
```

Comment:
```suggestion
    timeElapsed: number;
```
Based on https://github.com/facebook/jscodeshift/blob/7e3afb514d27ace2b00dc43695c8086498ab4672/src/Runner.js#L293